### PR TITLE
update message address on the zmq side

### PR DESCRIPTION
**Pull request by thraxil**
need routing address to match up with what the websockets side is using

### CODE REVIEW

**Comment by nikolas** [comment]
Strange, the behave tests both stopped at the same point with this change:
https://travis-ci.org/ccnmtl/uelc/builds/183938185#L1122
https://travis-ci.org/ccnmtl/uelc/builds/183938066#L1114

I'm restarting both tests to make sure it's not a fluke.

**Comment by thraxil** [comment]
It's happening in the same place on both and that happens to be right after a submit, which involves hitting the zmq socket stuff. `uelc/main/views.py` has `behave_socket_open()` and `behave_socket_recv()` functions that apparently are doing something to prevent the socket from hanging on behave tests.

I just haven't figured out yet how changing the address internal to the message would change anything with the socket interaction.

File: uelc/main/views.py [hunk 1]
```diff
@@ -467,7 +467,7 @@ def notify_facilitator(self, request, section, user, msg):
             notification=notification)
 
         e = dict(address="%s.pages/%s/facilitator/" %
-                 (settings.ZMQ_APPNAME, section.hierarchy.name),
+                 (settings.ZMQ_APPNAME, "module_%02" % section.hierarchy.pk),
```
Comment (sdreher) [inline uelc/main/views.py]: missing `d` at the end of the 02 format string

Comment (sdreher) [inline uelc/main/views.py]: This update is also needed in a few other spots. I'll add a commit.

**Pull request merged**